examples/sil/carbon-aware_control_unit/main.py

```diff
@@ -33,5 +33,5 @@ def argparser() -> argparse.ArgumentParser:
 if __name__ == "__main__":
     parser = argparser()
     args = parser.parse_args()
-    cacu = CarbonAwareControlUnit(args.server_address, args.node_ids)
+    cacu = CarbonAwareControlUnit(args.server_address, args.node_ids[0])
```

Comment:
does this input the whole list that was given or only the first list item from the cli?

Reply:
I see, thank you!